src/main/java/qna/domain/Answer.java

```diff
@@ -10,12 +10,17 @@
 public class Answer extends BaseEntity {
     @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private Long writerId;
-    private Long questionId;
     @Column(columnDefinition = "longtext")
     private String contents;
     private boolean deleted = false;
 
+    @JoinColumn(foreignKey = @ForeignKey(name = "fk_answer_to_question"))
```

Comment:
요구사항 매핑 잘 해주셨습니다 👍 

도메인엔티티명.PK로 컬럼명이 자동 부여 되겠지만
그래도 명시적인게 좋아서 컬럼 명을 표기하는 편입니다 😄 

```suggestion
    @JoinColumn(name = "question_id", foreignKey = @ForeignKey(name = "fk_answer_to_question"))
```


이 부분은 참고만 부탁 드리겠습니다 🙇